aws/lambda/lambda_function.py

```diff
@@ -37,9 +37,10 @@ def handler(event, context):
     if not article.is_valid_body():
         return {
             "statusCode": 400,
```

Comment:
400 is bad request. I think this case should still be considered a valid request since nothing about the request is actually incorrect. We can choose a different status code to indicate the request is successful but different response body is expected. In my opinion status code 202 (Accepted) seems suitable.

`The request has been accepted for processing, but the processing has not been completed`

Reply:
Great point! Updated the status code.